airflow/_vendor/connexion/operations/openapi.py

```diff
@@ -197,7 +197,7 @@ def example_response(self, status_code=None, content_type=None):
         except (KeyError, IndexError):
             pass
         try:
-            return (deep_get(self._responses, example_path), status_code)
+            return deep_get(self._responses, example_path), status_code
```

Comment:
I’d prefer keeping this, or split this into two lines

```python
response = deep_get(self._responses, example_path)
return response, status_code
```

otherwise it is too easy to miss this is actually returning a tuple.

Reply:
I'll revert it as it was.